expression/builtin_string_vec.go

```diff
@@ -1534,13 +1534,52 @@ func (b *builtinReplaceSig) vecEvalString(input *chunk.Chunk, result *chunk.Colu
 }
 
 func (b *builtinMakeSetSig) vectorized() bool {
-	return false
+	return true
 }
 
 // evalString evals MAKE_SET(bits,str1,str2,...).
 // See https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_make-set
 func (b *builtinMakeSetSig) vecEvalString(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	nr := input.NumRows()
+	bitsBuf, err := b.bufAllocator.get(types.ETInt, nr)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(bitsBuf)
+	if err := b.args[0].VecEvalInt(b.ctx, input, bitsBuf); err != nil {
+		return err
+	}
+
+	strBuf := make([]*chunk.Column, len(b.args)-1)
```

Comment:
we can use one buf to store arg1, arg2...
refer to this PR
https://github.com/pingcap/tidb/pull/13268

Reply:
I tried this kind of method some days before and found it would be slower than the non-vectorzied version sometimes just like in https://github.com/pingcap/tidb/pull/13268

in https://github.com/pingcap/tidb/pull/13268, the performance of vectorized version is not good. It gets about 10% slower.
```
BenchmarkVectorizedBuiltinStringFunc/builtinConcatSig-VecBuiltinFunc-2              6320            233318 ns/op          411024 B/op       2710 allocs/op
BenchmarkVectorizedBuiltinStringFunc/builtinConcatSig-NonVecBuiltinFunc-2           5307            217145 ns/op          160848 B/op       3732 allocs/op
```

I will take a try in my PR to test if it will get slower or faster to use only one buf.